src/invoices/InvoiceDetails/InvoiceActions/InvoiceActions.js

```diff
@@ -48,13 +50,14 @@ const InvoiceActions = ({
         </Button>
       </IfPermission>
       {
-        !isApprovePayEnabled && invoiceLinesCount > 0 && isPayable(invoice.status) && (
+        !isApprovePayAvailable && invoiceLinesCount > 0 && isPayable(invoice.status) && (
           <IfPermission perm="invoice.item.pay">
             <Button
               data-testid="invoice-pay"
               data-test-invoice-action-pay
               buttonStyle="dropdownItem"
               onClick={onPay}
+              disabled={isDisabled}
```

Comment:
isApprovePayEnabled ?

Reply:
I think `isDisabled` is more extensible and it will prevent code redundancy because we are using it in 3 places(Line 60, 77, 97)

Reply:
in 3 places you can use `!isApprovePayEnabled`, I'm for YAGNI